src/scales/interpolatedColorScale.ts

```diff
@@ -14,6 +14,7 @@ export module Scale {
    * By default it generates a linear scale internally.
    */
   export class InterpolatedColor extends AbstractScale<number, string> {
+    protected _d3Scale: D3.Scale.QuantitativeScale;
```

Comment:
It was required when I tried to put a `ticks()` call on `Scale.InterpolatedColor`, but not necessary any more.